index.html

```diff
@@ -86,20 +86,20 @@
         <br>
 
 	     <section class="row text-center">	
-            <div class=" col-md-2 col-sm-6">
-                <img class="img-responsive" src="{{ site.baseurl }}/img/ucr.jpg" alt="ucr" />
+            <div class="col-md-2 col-sm-6">
+                <center><img class="img-responsive" src="{{ site.baseurl }}/img/ucr.jpg" alt="ucr" /></center>
```

Comment:
Huh, I didn't even know <center> was a tag -- although it [looks](https://www.w3schools.com/tags/tag_center.asp) like it's deprecated now?